src/game/game.go

```diff
@@ -0,0 +1,36 @@
+package game
+
+type Grid [][]int
```

Comment:
I would use bytes instead of `int`. Let's save memory, let's save energy, let's save polar bears!

Reply:
And cats!